### PR TITLE
Request buffering tests

### PR DESCRIPTION
**Pull request by PidgeyBE**
### Checklist

- [ ] The Pull Request has tests
- [ ] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

* [Implement ...]

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -9,4 +9,5 @@ RUN apt-get install -y \
         unzip \
         git \
         m4 \
-        libyaml-dev
+        libyaml-dev \
+        libcurl4-openssl-dev
```
Comment (StarlightIbuki) [inline .devcontainer/Dockerfile:13]: I think we don't need to introduce a dependence for the release if it's only used in tests.
I'm not familiar with the build process and this looks like a part of it to me.

File: .devcontainer/docker-compose.yml [hunk 2]
```diff
@@ -35,6 +35,8 @@ services:
       KONG_PG_HOST: db
       OPENSSL_DIR: /usr/local/kong
       CRYPTO_DIR: /usr/local/kong
+      # env vars to be able to run tests
+      KONG_TEST_PG_DATABASE: kong
```
Comment (StarlightIbuki) [inline .devcontainer/docker-compose.yml:39]: Same here.

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (PidgeyBE) [reply .devcontainer/Dockerfile:13]: Hey, thanks for the review!
This Dockerfile is not used for releases though! It is the `.devcontainer`/Dockerfile. It's used for development: https://code.visualstudio.com/docs/devcontainers/containers
For me it was the only way I could get the kong stack running locally to run the integration tests on my own PC.

So it's not part of the kong build process.

File: .devcontainer/docker-compose.yml (hunk 2 above)
Comment (PidgeyBE) [reply .devcontainer/docker-compose.yml:39]: ~Same comment, this is part of VSCode's `.devcontainer` setup.
I think running tests is part of development.

**Pull request merged**